File: typescript/Structural/10-Facade/Subsystem/JobCardRegistry.ts
export class JobCardRegistry {
  private jobs: { jobId: string; title: string; status: string }[] = [];

  register(jobId: string, title: string): void {
    this.jobs.push({ jobId, title, status: "Active" });
    console.log(`  🗂️  JobCardRegistry: Job #${jobId} '${title}' registered.`);
  }

  get count(): number {
    return this.jobs.length;
  }
}

File: csharp/Structural/10-Facade/Subsystem/JobCardRegistry.cs
namespace Facade.Subsystem;

public class JobCardRegistry
{
    private readonly List<(string JobId, string Title, string Status)> _jobs = new();

    public void Register(string jobId, string title)
    {
        _jobs.Add((jobId, title, "Active"));
        Console.WriteLine($"  🗂️  JobCardRegistry: Job #{jobId} '{title}' registered.");
    }

    public int Count => _jobs.Count;
}
